content/app/github/window-helper.js

```diff
@@ -0,0 +1,41 @@
+define("content/app/github/window-helper", function(require, exports, module) {
+
+	// imports
+	var Box = require("content/app/models/box");
+
+	/**
+	 * Helper for accessing and controlling window and root page elements.
+	 */
+	var WindowHelper = Class(Object, {
+
+		$statics : {
+
+			/**
+			 * Get viewport bounding box.
+			 * @return {Box}
+			 */
+			getViewportBoundingBox : function () {
+				var width = document.documentElement.clientWidth;
+				var height = document.documentElement.clientHeight;
+				var left = document.body.scrollLeft;
+				var top = document.body.scrollTop;
+				return new Box(left, top, width, height);
+			},
+
+			/**
+			 * Get viewport bounding box relative to client visible area.
+			 * That means that viewport is at position (0,0).
+			 * @return {Box}
+			 */
+			getViewportClientBoundingBox : function () {
+				var bbox = WindowHelper.getViewportBoundingBox();
+				return bbox.move(0, 0);
```

Comment:
As I explained in Box comments, move implemented in this way does not make much sense. I would also not use getViewportBoudingBox here, for me it would be much more readable to do the following:
var width = ....
var height = ...
return new Box(0, 0, width, height)